src/main/java/dk/cngroup/lentils/controller/HintController.java

```diff
@@ -23,18 +26,25 @@
     private static final String VIEW_HINT_LIST = "hint/list";
     private static final String VIEW_HINT = "hint/detail";
     private static final String REDIRECT_HINT_LIST = "redirect:/admin/hint";
+    private static final String CYPHERID_PARAMETER = "?cypherId=";
+    private static final String HEADING_NEW_HINT = "Nový hint";
+    private static final String HEADING_EDIT_HINT = "Upravit hint";
 
     private static final String TEMPLATE_ATTR_CYPHER = "cypher";
     private static final String TEMPLATE_ATTR_HINT = "hint";
     private static final String TEMPLATE_ATTR_HEADING = "heading";
 
     private CypherService cypherService;
```

Comment:
Netýká se přímo tvé změny, ale když už to vidím a měníš to tady, můžeš tyhle 3 atributy označit jako final? :) Díky